docs/source/cpp/compute.rst

```diff
@@ -420,39 +420,39 @@ precisions/scales will be promoted appropriately. Mixed decimal and
 floating-point arguments will cast all arguments to floating-point, while mixed
 decimal and integer arguments will cast all arguments to decimals.
```

Comment:
Should we add a note about temporal casting here?

Reply:
Will do.

Reply:
Done.